index.html

```diff
@@ -291,5 +291,29 @@ <h5>Skills</h5>
           </div>
         </article>
       </section>
-		</body>
+      <section class="contact con-fnt">
+        <div class="cont_frm_shapes-mob"></div>
+          <header class="con-bg-ls">
+          <h1 class="con-txt-hd">Contact me</h1>
+            <p class="con-txt-bd">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+          </p>
+          </header>
+          <form>
+            <ul>
+              <li id="usr-txtarea">
+                <input type="text" name="name" placeholder="Enter Full Name" maxlength="15" class="usr-mail-txt usr-cont frm-lb" required="required" />
```

Comment:
- [ ] The maxlength should be 30 as required.

[OPTIONAL] You don't need to add required="required" just write `required` and it would also work.